deployer/utils/file_acquisition.py

```diff
@@ -241,27 +240,14 @@ def get_decrypted_files(files):
         yield [stack.enter_context(get_decrypted_file(f)) for f in files]
 
 
-def get_all_cluster_yaml_files(is_test=False):
+def get_all_cluster_yaml_files():
     """Get a set of absolute paths to all cluster.yaml files in the repository
 
-    Args:
-        is_test (bool, optional): A flag to determine whether we are running a test
-            suite or not. If True, only return the paths to cluster.yaml files under the
-            'tests/' directory. If False, explicitly exclude the cluster.yaml files
-            nested under the 'tests/' directory. Defaults to False.
-
     Returns:
         set[path obj]: A set of absolute paths to all cluster.yaml files in the repo
     """
-    # Get absolute paths
-    if is_test:
-        # We are running a test via pytest. We only want to focus on the cluster
-        # folders nested under the `tests/` folder.
-        return set(REPO_ROOT_PATH.glob("tests/test-clusters/**/cluster.yaml"))
-
-    # We are NOT running a test via pytest. We only care about the clusters under config/clusters
     return {
         path
-        for path in REPO_ROOT_PATH.glob("config/clusters/**/cluster.yaml")
+        for path in CONFIG_CLUSTERS_PATH.glob("**/cluster.yaml")
```

Comment:
The key change is that letting the deployer reference a constant to what path to glob cluster.yaml files from, and then have that constant be mocked during tests.